dogm/demo/utils/test/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+ADD_EXECUTABLE (utils_spec simulator_spec.cpp)
```

Comment:
To fix the windows build you should add `SET(gtest_force_shared_crt ON CACHE BOOL "" FORCE)` here.
See https://github.com/google/googletest/blob/master/googletest/README.md#visual-studio-dynamic-vs-static-runtimes

Reply:
Done